.github/workflows/mifos-ci.workflow.yml

```diff
@@ -0,0 +1,54 @@
+name: Mifos CI
+
+on:
+  pull_request:
+    branches: [ master, staging, dev ]
+
+jobs:
+  build:
+    name: CI
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
No need for a matrix, use the same explicit version used in the workflow that publishes to s3